.github/docker/Dockerfile.ci-image

```diff
@@ -13,14 +13,11 @@ RUN set -eux; \
     apt-get update; \
     apt-get install -y --no-install-recommends \
       ca-certificates wget curl git zstd gcc libc6-dev \ 
-      nodejs npm postgresql-client python3.9 python3-pip \
-      unzip; \
+      nodejs npm postgresql-client python3.11 python3-pip \
+      unzip lld awscli g++-aarch64-linux-gnu libc6-dev-arm64-cross \
+      ; \
```

Comment:
:art: unnecessary newline
```suggestion
      unzip lld awscli g++-aarch64-linux-gnu libc6-dev-arm64-cross; \
```

Reply:
I put it on a new line to avoid having to move it every time we add something. To me is like having a closing parenthesis on a new line but I don't have a strong opinion on this

Reply:
i see, up to you to decide